todo/db.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+
+from . import db
+
+
+class Todo(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    task = db.Column(db.String(120))
+    done = db.Column(db.Boolean)
+    date = db.Column(db.DateTime)
+
+    def __init__(self, task):
+        self.task = task
+        self.done = False
+        self.date = datetime.datetime.now()
+
+
+db.create_all()
```

Comment:
It's not production safe to run this on every import

Reply:
You can do smth like this
```
heroku run python3
>>> from app import db
>>> db.create_all()
```

Reply:
from todo import db
db.db.create_all()